src/test/java/com/puppycrawl/tools/checkstyle/DetailAstImplTest.java

```diff
@@ -693,10 +694,15 @@ public void testManyComments() throws Exception {
             bw.write("}\n");
         }
 
+        final int initialFileCount = Objects.requireNonNull(temporaryFolder.list()).length;
+
         final DefaultConfiguration checkConfig = createModuleConfig(TodoCommentCheck.class);
 
-        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkConfig, file.getAbsolutePath(), expected);
+        execute(checkConfig, file.getAbsolutePath());
+
+        final int finalFileCount = Objects.requireNonNull(temporaryFolder.list()).length;
+
+        Assertions.assertEquals(initialFileCount, finalFileCount);
```

Comment:
No, this is too crazy, next engineer will not understand this.
Let's add header to file and use our verify with embedded config.

Reply:
There doesn't exist any file with the name `InputDetailASTManyComments`. 
I tried running the test with verifyWithInlineConfigParser and it mentioned the following file path - Config comment not specified properly in /var/folders/ry/ztsc86hs7wvc09v5qtvt1gtw0000gn/T/junit13776525294712844995/InputDetailASTManyComments.java

which is again nowhere to be found in the project.

Reply:
Yes, this file is generated in this test. See the for loop above this code. We need to update this test to insert our config comment into this generated file.

Reply:
Yes done